README.md

```diff
@@ -55,8 +115,16 @@ To run the pipeline using example data provided in the repository:
 
 This command uses test data to demonstrate the pipeline's functionality, ideal for initial testing and familiarization.
 
+# Result
+
 ## Note:
-The default output (--o) dataframe is stored in Parquet format due to its efficient storage, faster data access, and enhanced support for complex data structures. The Parquet file can be accessed in several ways:
+* The default output (--o) dataframe is stored in Parquet format due to its efficient storage, faster data access, and enhanced support for complex data structures. 
```

Comment:
I think `--o` should be either (or both) `-o`, `--output`

Reply:
Thanks! updated it.